src/web_backups/main/ui/createConnection/CreateConnection.java

```diff
@@ -0,0 +1,30 @@
+package web_backups.main.ui.createConnection;
+
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.JSchException;
+import com.jcraft.jsch.Session;
+
+
+public class CreateConnection {
+    public static void connect(String username, String host,int port, String password) {
+        try {
+            JSch jsch = new JSch();
+            Session session = jsch.getSession(username, host, port);
+            session.setPassword(password);
+            session.setConfig("StrictHostKeyChecking", "no");
+            session.setTimeout(15000);
+            session.connect();
+        } catch (JSchException e) {
```

Comment:
Please add a validation message for the user. Use NoValidDataException.java instead.